lib/test.py

```diff
@@ -9,15 +8,17 @@
 from functools import partial
 from hashlib import md5
 
+import gevent
```

Comment:
From https://github.com/tarantool/tarantool/wiki/Code-review-procedure#general-coding-points-to-check : "Don't do a change if it is not related to the patch."
Shuffling the import order is not related to the problem solved in the patch. Here and below.

Reply:
Got it
Thank you